src/lib/storage/segment_access_statistics.cpp

```diff
@@ -0,0 +1,171 @@
+#include "segment_access_statistics.hpp"
+
+namespace opossum {
+
+std::chrono::time_point<std::chrono::steady_clock> AtomicTimedAccessStrategy::start_time = std::chrono::steady_clock::now();
```

Comment:
Dient als Referenzzeitpunkt. Alle Zeitpunkte werden ab start_time gemessen.

Reply:
Not neccessary anymore. Will be managed externally (like the AntiCachingPlugin).